stacks/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@double-agent/stacks",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "downloadRunnerData": "yarn workspace @double-agent/external-data downloadRunnerData",
+    "0": "node -r source-map-support/register scripts/0-collectFoundationalProfiles.js",
+    "1": "node -r source-map-support/register --max-old-space-size=10000 scripts/1-extractFoundationalProbes.js",
+    "2": "node -r source-map-support/register scripts/2-collectUserAgentsToTest.js",
+    "3": "node -r source-map-support/register scripts/3-runAssignments.js",
+    "3-hero": "yarn 3",
+    "3-secretAgent": "node -r source-map-support/register scripts/3-runAssignments.js --runner=secret-agent",
+    "3-puppeteer": "node -r source-map-support/register scripts/3-runAssignments.js --runner=puppeteer",
+    "4": "node -r source-map-support/register scripts/4-analyzeAssignmentResults.js"
+  },
+  "dependencies": {
+    "@secret-agent/core": "^1.6.4",
```

Comment:
Looks like you're missing the double-agent dependencies. We added some eslint configs (monorepo-cop) to check when you reference dependencies not actually installed. I'm not 100% sure it would matter for stacks, since I don't think we would ever publish that to npm. But it might be helpful to think through how you would actually use this as a library over npm. 

- I think we would want to have external-data as a dependency, or at list a "bin" script that you can execute. Kind of like how browserlist updates their data.
- I think we'd need to publish runner, but how would you actually use the rest of this? Or are you imagining this being accessed via the workspaces?

Reply:
> Looks like you're missing the double-agent dependencies. We added some eslint configs (monorepo-cop) to check when you reference dependencies not actually installed

Good point going to look into it and address it appropriately.

> I think we would want to have external-data as a dependency, or at list a "bin" script that you can execute. Kind of like how browserlist updates their data.

Sounds like a good start already.

> I think we'd need to publish runner, but how would you actually use the rest of this? Or are you imagining this being accessed via the workspaces?

Yes indeed. I would only publish runner, similar to how other double-agent packages are published. Stacks I wouldn't publish indeed, as it is more of an example as well as could it serve as your own test stack.

Personally I probably end up using the workspaces approach for the long time, but only because I imagine that it won't be the last time that I run a forked version just to keep up to date with my own proposals and the edge of it all. However we should definitely ensure that people can use it like any other npm library indeed. Which I believe this patch, if done right, should allow, right?

Reply:
>  it continues to make me wonder when we should let you into Slab (or just open it as is) so you can see what's going on there, and possibly how it could fit together.

Up to you. I don't mind either way. For now I'm busy enough as is in getting my code that integrates secret-agent to a state where it's useful enough as well as still needing to actually be able to use double-agent as-is (with this PR). So not yet blocked on needing to be able to generate the input/verification data myself. But at some point soon I suppose that does become a concern of mine. Would be great to document that process wel and provide tooling where possible for reuse by others. And next to that I wouldn't mind to help where I can on keeping that data up to date.

Reply:
> One thing I am interested in is exploring what it looks like for the "data" to live in its own repo, which would be imported here a submodule.

If you talk about the cleaned up data than that can be a thing, but if you talk about all that raw data than that really is a bit nuts I would think.

The other idea you had regarding simply having a script that can get it from somewhere, including just the latest version, could be something as well. But that's kinda what we have today already.

Reply:
Added the dependencies: https://github.com/ulixee/double-agent/pull/62/commits/f89c89045500b4833fb0099c38b464658ae866d2

Did it by manually checking what dependencies I use, because if I run `eslint` (with all eslint plugins installed) from root, it doesn't seem to do anything? Anyway, I believe this should fix the dependencies issues.

Reply:
Thanks @GlenDC. For what it's worth, I don't think we have monorepo-cop setup on DoubleAgent. It's in SecretAgent eslint setup.